admin-dev/themes/new-theme/template/components/layout/quick_access.tpl

```diff
@@ -22,11 +22,11 @@
         data-icon="{$quick_access_current_link_icon}"
         data-url="{$link->getQuickLink($smarty.server['REQUEST_URI']|escape:'javascript')}"
         data-post-link="{$link->getAdminLink('AdminQuickAccesses')}"
-        data-prompt-text="{l s='Please name this shortcut:' js=1}"
+        data-prompt-text="{l s='Please name this shortcut:' js=1 d='Admin.Navigation.Header'}"
         data-link="{$quick_access_current_link_name|truncate:32}"
       >
         <i class="material-icons">remove_circle_outline</i>
-        {l s='Remove from QuickAccess'}
+        {l s='Remove from QuickAccess' d='Admin.Navigation.Header'}
```

Comment:
What about _Delete_ in _Admin.Actions_, or at least _Remove from Quick Access_ with a space between the two words (I don't know why there is no space in the current string)?

Reply:
In _Admin.Actions_ please. ;-)